spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  subject { FactoryBot.create(:recipe, public: true) }
+
+  before { subject.save }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid without a name' do
+    subject.name = nil
+    expect(subject).not_to be_valid
+  end
+
+  it 'is not valid without a preparation time' do
+    subject.preparation_time = nil
+    expect(subject).not_to be_valid
+  end
+
+  it 'is not valid without a cooking time' do
+    subject.cooking_time = nil
+    expect(subject).not_to be_valid
+  end
+
+  it 'is not valid without a description' do
+    subject.description = nil
+    expect(subject).not_to be_valid
+  end
+
+  it 'is not valid without a user_id' do
+    subject.user_id = nil
+    expect(subject).not_to be_valid
+  end
+
+  it 'is not valid without a boolean value for public' do
+    subject.public = nil
+    expect(subject).not_to be_valid
+  end
```

Comment:
- [ ] This test failed when I ran the testing command. Could you work on it so all the tests implemented pass as expected? 👍🏽

<img width="790" alt="Screenshot 2023-11-25 at 10 34 39 AM" src="https://github.com/ulises2607/rails-recipe-app/assets/50941074/51b6f425-fe29-45ec-8fdf-eb7844b475b1">

Reply:
Hi @AmaduKamara thank you for the keen eye I forgot to check the test my bad